tests/browser-detector/scripts/build.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+error_reporting(E_ERROR | E_WARNING | E_PARSE);
+
+$allTests = [];
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$jsonParser = new \Seld\JsonLint\JsonParser();
+
+$finder = new \Symfony\Component\Finder\Finder();
+$finder->files();
+$finder->name('*.json');
+$finder->ignoreDotFiles(true);
+$finder->ignoreVCS(true);
+$finder->sortByName();
+$finder->ignoreUnreadableDirs();
+$finder->in(__DIR__ . '/../vendor/mimmi20/browser-detector-tests/tests/issues');
+
+foreach ($finder as $file) {
+    /** @var \Symfony\Component\Finder\SplFileInfo $file */
+    if (!$file->isFile() || 'json' !== $file->getExtension()) {
+        continue;
+    }
+
+    $filepath = $file->getPathname();
+
+    $content = file_get_contents($filepath);
+
+    if ('' === $content || PHP_EOL === $content) {
+        continue;
+    }
+
+    try {
```

Comment:
This test suite is pretty large and takes a while to generate. I wonder if we should implement some sort of caching of the output that's generated, so any subsequent runs just use the already built file (if the version is the same, or some other sort of invalidation).

Reply:
I will remove this test suite, as it runs into a `out of memory` error. I was thinking about changing the test/parse commands to be able to handle single useragents, but this would need more time.